package.json

```diff
@@ -83,8 +83,7 @@
     "requirejs": "gulp requirejs",
     "generateDocumentation": "gulp generateDocumentation",
     "instrumentForCoverage": "gulp instrumentForCoverage",
-    "eslint": "gulp eslint",
-    "eslint-watch": "gulp eslint-watch",
+    "eslint": "eslint ./**/*.js ./**/*.html --cache",
```

Comment:
OK, I misunderstood what "warnings" were.  I thought there was a class of eslint rules but it looks like it's just eslint being all "are you sure".  I don't see the warning you're describing locally, but I do on travis.  I'll add the --quite flag.